core/templates/pages/learner-group-pages/create-group/add-syllabus-items.component.html

```diff
@@ -40,14 +40,13 @@
       </form>
     </div>
 
-    <div class="oppia-same-row-container">
+    <div class="oppia-same-row-container oppia-search-filter">
       <div ngbDropdown class="search-bar-float-left oppia-search-bar-type-selector dropdown" autoClose="outside">
         <button ngbDropdownToggle
                 (click)="openSubmenu($event, 'type')"
                 type="button"
                 class="btn oppia-search-bar-type-input dropdown-toggle"
                 title="{{ selectionDetails.types.description | translate }}">
-          <i *ngIf="isMobileViewActive()" class="material-icons md-18 type-selector-icon">&#xE574;</i>
```

Comment:
?

Reply:
these icons were added in a previous sub-milestone PR, but we do not need it anymore